tests/test_docker_images.py

```diff
@@ -0,0 +1,39 @@
+import docker
+from tests.helm_template_generator import render_chart
+import jmespath
+
+
+def test_docker_images(docker_client):
```

Comment:
wdyt to refactor it to pytest parametrized? so you can see each test for each image rather than doing for loop.

Reply:
This was done well

```
tests/test_docker_images.py::test_docker_image[quay.io/astronomer/ap-nats-server:2.7.2-1]
```